common/models/types/index.d.ts

```diff
@@ -216,6 +216,14 @@ declare interface Suggestion {
    */
   transformId?: number;
 
+  /**
+   * A unique identifier for the Suggestion itself, not shared with any others -
+   * even for Suggestions sourced from the same Transform. 
+   * 
+   * The lm-layer is responsible for setting this field, not models.
+   */
+  id?: number;
```

Comment:
Is this more of a guid/uuid? Just to avoid confusion with all the other id's we have around (Transform.id?)

Reply:
It is not.  This one is rooted on the new `SUGGESTION_ID_SEED` from `ModelCompositor`.